test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +47,9 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        assertTrue(Utils.isAnyNull((Object)null));
+    }
```

Comment:
You do not have to include the changes from last week. Please provide each PR on a separate branch. Do not include previous changes.